app/actions/remote/post.ts

```diff
@@ -900,6 +900,7 @@ export const markPostAsUnread = async (serverUrl: string, postId: string) => {
                 let totalMessages = channel.total_msg_count;
                 let messages = channelMember.msg_count;
                 let mentionCount = channelMember.mention_count;
+                const urgentMentionsCount = channelMember.urgent_mention_count || 0;
```

Comment:
Why would it be undefined? Older servers?

Also there is no urgent_mention_root as the other ones? I'm almost sure urgent only applies to root posts but just to confirm

Reply:
Yup, nice catch, it shouldn't be optional. Changed it.